### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by MessiasLima** [Chozzle/compose-macos-theme#8]

**Comment by Chozzle** [comment]
Not currently. It was a project for my own enjoyment, but to really keep going it would need people to actually use it in production. I'm happy to transfer ownership to whoever wants it

**Comment by MessiasLima** [comment]
This project was mentioned in the last KotlinConf. https://www.youtube.com/watch?v=Mgf_9kxM1BA
It would be nice if it didn't die

**Comment by Chozzle** [comment]
I totally agree. Happy to help if people want to use it and transfer ownership if someone is wants to use this in a production app.

**Comment by woj-tek** [comment]
I just found the project and just wanted to say that IMHO it would be awesome if Jetbrains incorporated it into Compose itself offering more native-looking L&F on desktop :)

**Comment by kdroidFilter** [comment]
@woj-tek you can check my new project https://kdroidfilter.github.io/compose-macos-26-ui/#home but it's free only for oss
